src/components/S/Base.vue

```diff
@@ -1,11 +1,22 @@
 <script setup>
-defineProps({ definition: Object });
+defineProps({ definition: Object, error: String });
 </script>
 
 <template>
   <div>
     <label v-if="definition.title" v-text="definition.title" class="s-label" />
-    <slot />
-    <legend v-if="definition.description" v-text="definition.description" />
+    <div>
+      <slot />
+      <div
+        :class="[
+          'bg-pink-100 text-red text-sm relative -z-10 transition-all duration-200 flex items-center px-3 pb-1 pt-4 rounded-b-3xl',
```

Comment:
The class / style should not be defined here but on the scss file, this could have just a .v-error class and v-error would apply all the styling classes, this way style stay customisable if used on others UI.